src/main/java/baseball/NumberRepository.java

```diff
@@ -0,0 +1,55 @@
+package baseball;
+
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ * @ClassName NumberRepository
+ * 
+ * @Version 0.0.0
+ * 
+ * @Date 20191203
+ * 
+ * @author asqwklop12
+ * 
+ * @EMAIL asqwklop12@gmail.com
+ *
+ */
+
+public abstract class NumberRepository {
+
+	private static final int REPOSITORYSIZE = 3;
```

Comment:
```suggestion
	private static final int REPOSITORY_SIZE = 3;
```

보통 상수를 사용할 때 여러 단어들을 사용해야 할 경우 각각의 단어 사이를 언더바(`_`) 로 구분하는 것이 자바 컨벤션입니다.